clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10274,6 +10274,11 @@ def warn_array_comparison : Warning<
   "to compare array addresses, use unary '+' to decay operands to pointers">,
   InGroup<DiagGroup<"array-compare">>;
 
+def warn_array_comparison_cxx26 : Warning<
+  "comparison between two arrays compare their addresses not their contents; "
```

Comment:
I wonder if this should just say 'is ill formed in C++26' or something instead of just repeating the same message?

Reply:
I agree, "comparison between two arrays is ll-formed in C++26; " will be better in this case

Reply:
I would prefer "not allowed" instead of "ill-formed".

Example:
 https://github.com/llvm/llvm-project/blob/02ad623bb560afa1a789b49f715c9a0e48ea9b16/clang/include/clang/Basic/DiagnosticSemaKinds.td#L665-L667

Reply:
I found that we have diagnostics with 'not allowed' and 'ill-formed' in the codebase, I am okay with both, and we can agree on opinion.

What do you think the best option here?

@erichkeane @cor3ntin @zwuis

Reply:
I don't think it matters a lot. ill formed conveys it's a standard requirement pretty well